### ISSUE TITLE
Auto complete issue in thunderbird

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#181]
**Reported by chris1988 on 2 Jun 2008 08:00 UTC**
When in a compose window in thunderbird nvda currently doesn't read the name of an address automatically.  The Autocomplete doesn't read.

**Comment by nvdaTrac** [comment]
**Comment by pvagner on 2 Jun 2008 12:12 UTC**
I think this is a core not virtual buffers issue. We were also discussing a problem with auto complete in mozilla firefox that once the list of possible auto completions is displayed NVDA is unable to read the options. I am not sure if a ticket has already been created for this.
Also I think Jamie has commented somewhere that in thunderbird 3 they are still using some older component for addresses in the composition window so perhaps this is expected to be changed in the mean time.
Surprisingly now My copy of firefox does not report an edit field at the google.com frontpage as autocomplete so I don't know if that are my settings or something has been changed.

**Comment by nvdaTrac** [comment]
**Comment by mdcurran on 7 Jun 2008 07:07 UTC**
Fixed in 2106. Selections in IAccessible2 text is now detected automatically like it is for standard Windows edit fields. As the autoComplete control in thunderbird selects the completion text, this now speaks as it appears.
